packages/tldraw/src/lib/tools/SelectTool/childStates/PointingShape.ts

```diff
@@ -33,7 +34,11 @@ export class PointingShape extends StateNode {
 			outermostSelectingShape.id === focusedGroupId ||
 			// ...or if the shape is within the selection
 			selectedShapeIds.includes(outermostSelectingShape.id) ||
-			this.editor.isAncestorSelected(outermostSelectingShape.id) ||
+			// ...or if the selected ancestor doesn't allow select on enter
+			(selectedAncestor &&
+				!this.editor
+					.getShapeUtil(selectedAncestor)
+					.canSelectChildOnPointerDownWhileSelected(selectedAncestor, outermostSelectingShape)) ||
```

Comment:
If this is just a reference to the note shape, let's hardcode it. I'm trying to not add more stuff to the public API under we either need it for two different things or someone else wants to use it too.

Reply:
done!